_component_design/tables-basic.md

```diff
@@ -67,6 +67,8 @@ Tables are the bread-and-butter component for displaying complex objects in a ne
 
```

Comment:
Update status and last-mod date

Reply:
I promise I will remember this someday. Updated.